examples/asr/asr_ctc/README.md

```diff
@@ -0,0 +1,30 @@
+# ASR with CTC Models
+
+This directory contains example scripts to train ASR models using Connectionist Temporal Classification Loss. 
+
+Currently supported models are - 
+
+* Character based CTC model
+* Subword based CTC model
+
+# Model execution overview
+
+The training scripts in this directory execute in the following order. When preparing your own training-from-scratch / fine-tuning scripts, please follow this order for correct training/inference.
+
+```mermaid
+
+graph TD
+    A[Hydra Overrides + Yaml Config] --> B{Config}
+    B --> |Init| C[Trainer]
+    C --> D[ExpManager]
+    B --> D[ExpManager]
+    C --> E[Model]
+    B --> |Init| E[Model]
+    E --> |Constructor| F(Change Vocabulary)
+    F --> G(Setup Train + Validation + Test Data loaders)
+    G --> H(Setup Optimization)
+    H --> I[Maybe init from pretrained]
```

Comment:
Should this step be right after initializing the model? Or should it be the last thing?

Reason I ask is b/c wouldn't the `maybe_init_from_pretrained()` method override the previous behavior in nodes F/G/H?

Reply:
Maybe init from pretraind only loads the pytorch weights of the old file into the new file, it has no effect at all on any of the dataloaders / optim etc.